README.md

```diff
@@ -52,14 +52,18 @@ Install the MAU bundle (or, starting a dev session every time)
     git checkout master
     git pull
     bundle install
+    
+Note that on MacOS Mojave, you may need to run the following command in order for the `mysql` gem to be successfuly installed and the `bundle install` to complete. See https://gist.github.com/fernandoaleman/ee3ac6957c2ba4f7d7d33a251d58b191 for original instructions and for link to potential fix for the same issue on Catalina.
+
+    gem install mysql2 -v '0.5.3' -- --with-ldflags=-L/usr/local/opt/openssl/lib --with-cppflags=-I/usr/local/opt/openssl/include
 
 Tell Rails to build you an empty database
 
     rake db:create:all # only if you cloned the repo
 
-Tell Rails to setup that database and build a copy of that schema for testing
+Tell Rails to load the schema for the database and build a copy of that schema for testing
```

Comment:
```suggestion
Use the setup script to load the schema for database and then tell Rails to build a copy of that schema for testing
```

(This change should be paired with the comment below adding the line about using bin/setup)